src/ILCompiler.WebAssembly/src/CodeGen/ILToWebAssemblyImporter.cs

```diff
@@ -1022,7 +1022,26 @@ private void ImportCall(ILOpcode opcode, int token)
             if (opcode == ILOpcode.newobj)
             {
                 TypeDesc newType = callee.OwningType;
-                if (newType.IsString)
+                if (newType.IsArray)
+                {
+                    var paramCnt = callee.Signature.Length;
+                    var eeTypeDesc = _compilation.TypeSystemContext.SystemModule.GetKnownType("Internal.Runtime", "EEType").MakePointerType();
+                    LLVMValueRef dimensions = LLVM.BuildArrayAlloca(_builder, LLVMTypeRef.Int32Type(), BuildConstInt32(paramCnt), "newobj_array_pdims_" + _currentOffset);
+                    for (int i = paramCnt - 1; i >= 0; --i)
+                        LLVM.BuildStore(_builder, _stack.Pop().ValueAsInt32(_builder, true),
+                            LLVM.BuildGEP(_builder, dimensions, new LLVMValueRef[] { BuildConstInt32(i) }, "__pdims_ptr__"));
```

Comment:
It looks like this is indented incorrectly. Also the underscores around the name are inconsistent with other names

Reply:
So how should this line indent? As far as I remember I was using VSCode's default reformatting, though not so sure on this. I'm not on my working machine these days :(

Reply:
I misread this as two separate statements. That indentation is ok (but definitely makes the loop brackets important since this doesn't scan very easily otherwise).